components/organisms/filter_layout.vue

```diff
@@ -14,58 +14,58 @@
         </div>
         <div class="modal-content">
           <h2>Zoek verwerking</h2>
-          <form action="#result" @submit.prevent="submitFilter">
-            <div class="form-item">
-              <label for="name">Naam <span class="label-optional">(Optioneel)</span></label>
-              <input id="name" v-model="filter.name" type="text" name="name">
+          <div class="form-item">
+            <label for="name">Naam <span class="label-optional">(Optioneel)</span></label>
+            <input id="name" v-model="filter.name" type="text" name="name">
+          </div>
+          <checkbox-with-filter v-if="processors.length"
+                                :id="'services'"
+                                v-model="filter['processor[]']"
+                                :items="processors"
+                                :legend="'Verwerkende dienst'"
```

Comment:
How will we translate this labels?

Reply:
We won't. It's not a multi lingual site.